src/values.ts

```diff
@@ -1,21 +1,186 @@
+import { parse } from "path";
+import { ClientError } from "./errors";
+import * as PARSE from "./regex";
 /**
- * An Instant represents a fixed point in time (called "exact time"), without
- * regard to calendar or location, e.g. July 20, 1969, at 20:17 UTC.
+ * An wrapper around the Fauna `Time` type. It, represents a fixed point in time
```

Comment:
```suggestion
 * A wrapper around the Fauna `Time` type. It, represents a fixed point in time
```

OK to ship as is for now

Reply:
Why do I keep doing that?!